playbooks/roles/load-balancer-v2/files/haproxy.cfg.ctmpl

```diff
@@ -34,13 +34,6 @@ defaults
     timeout connect 5000
     timeout client  50000
     timeout server  2m
-    errorfile 400 /etc/haproxy/errors/400.http
-    errorfile 403 /etc/haproxy/errors/403.http
-    errorfile 408 /etc/haproxy/errors/408.http
-    errorfile 500 /etc/haproxy/errors/500.http
-    errorfile 502 /etc/haproxy/errors/502.http
-    errorfile 503 /etc/haproxy/errors/503.http
-    errorfile 504 /etc/haproxy/errors/504.http
```

Comment:
Did these exist at one time for any particular reason?

Reply:
Yeah I looked around to figure out how these were being created, and didn't see anything relevant even far back into the git history for the v1 role. I think most likely the package manager was installing them, but it didn't install them this time. I think we don't actually serve these anyway, we serve custom OpenCraft branded ones from nginx.

Reply:
Ah you know what, I hadn't run `apt-get purge` for haproxy which caused issues when re-installing which lead to these missing files. I'll add these back and purge and add back for all the new haproxy servers.

Reply:
Thanks for commenting on this.